mne/inverse_sparse/mxne_inverse.py

```diff
@@ -700,3 +730,119 @@ def tf_mixed_norm(evoked, forward, noise_cov,
         out = out, residual
 
     return out
+
+
+@verbose
+def compute_sure(M, gain, alpha_grid, sigma, n_mxne_iter, maxit, tol, n_orient, 
```

Comment:
```suggestion
def _compute_mxne_sure(M, gain, alpha_grid, sigma, n_mxne_iter, maxit, tol, n_orient, 
```
make this a private function with the `_` prefix